sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -13,34 +13,50 @@ Azure Cognitive Services Text Analytics is a cloud service that provides advance
 
 ### Prerequisites
 * An [Azure subscription][azure_sub].
-* An existing [Cognitive Services][cognitive_resource] or Text Analytics resource. If you need to create the resource, you can use the [Azure Portal][azure_portal] or [Azure CLI][azure_cli].
+* An existing Cognitive Services or Text Analytics resource.
 
-If you use the Azure CLI, replace `<your-resource-group-name>`, `<your-resource-name>`, `<location>`, and `<sku>` with your values:
+### Create a Cognitive Services or Text Analytics resource
+Text Analytics supports both [multi-service and single-service access][cognitive_resource_portal]. Create a Cognitive Services resource if you plan to access multiple cognitive services under a single endpoint/key. For Text Analytics access only, create a Text Analytics resource.
+
+You can create either resource using the [Azure Portal][cognitive_resource_portal] or [Azure CLI][cognitive_resource_cli]. Below is an example of how you can create a Text Analytics resource using the CLI:
```

Comment:
Let's do the Option 1 or Option 2 thing we talked about in the meeting today (see my PR)